prototype2/efu/HwCheck.h

```diff
@@ -31,7 +31,7 @@ class HwCheck {
   void setMinimumMTU(int mtu) { minimumMtu = mtu; }
 
   /// Gleaned from MacOS and CentOS and deemed ignore worthy
-  std::vector<std::string> defaultIgnoredInterfaces = {"ppp0", "docker", "ov-"};
+  std::vector<std::string> defaultIgnoredInterfaces = {"ppp0", "docker", "ov-", "wlp4s0"};
```

Comment:
This is not the intended use. Only those interfaces that we know for sure are irrelevant should be 
included. In stead use the -u 1500 option to the EFU which allows smaller MTUs

Reply:
I wasn't entirely clear on this -u business and I will first test that it actually fixes the problem for me. Whatever I am doing on my laptop, I am never ever receiving or sending anything via UDP or Kafka on the wifi interface, so why is the EFU adamant about checking up on how that interface is configured? Does it really need to check every interface on the machine?

Reply:
Yes the intent is to check mtu on all active, non-virtual, interfaces. In a office/laptop environment you never really know if you are using the wifi interface or the wired ethernet.

If the MTU is too small, packets will be fragmented int o 1500 byte packets upon transmission 
and reassembled upon reception. This overhead make performance comparisons unreliable and confusing.